addons/web/static/src/scss/form_view.scss

```diff
@@ -407,6 +407,10 @@ $o-form-label-margin-right: 0px;
                 padding-right: $o-statbutton-spacing;
                 line-height: 1.3;
 
+                .o_quick_editable {
```

Comment:
Hi @hop-odoo 
for the last point (the cursor issue) I don't think we should do this in this task. As this is going to impact every module, this should be done by the JS team.
It should be investigated a bit more. For example, in the following video, you can see that the cursor is a pointer when you hover over the attendance stat button but it is not clickable so the cursor should be the default cursor
https://drive.google.com/file/d/1bSigLNqcEYH7VkuheDP3F64AkokqE-Iw/view?usp=sharing

Reply:
@jbaudoux , Removed that point. Thanks